dev-libs/libuv/libuv-1.5.0.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit eutils autotools multilib-minimal
+
+DESCRIPTION="Cross-platform asychronous I/O"
+HOMEPAGE="https://github.com/libuv/libuv"
+SRC_URI="https://github.com/libuv/libuv/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD BSD-2 ISC MIT"
+SLOT="0/1"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux"
+IUSE="static-libs"
+
+DEPEND="
+	sys-devel/libtool
+	virtual/pkgconfig
+"
+
+src_prepare() {
+	echo "m4_define([UV_EXTRA_AUTOMAKE_FLAGS], [serial-tests])" \
+		> m4/libuv-extra-automake-flags.m4 || die
+
+	sed -i \
+		-e '/CC_CHECK_CFLAGS_APPEND(\[-g\])/d' \
+		configure.ac || die "fixing CFLAGS failed!"
```

Comment:
Doesn't this use some fancy autoconf caching, so you could pass `cv_something_something=no` instead? :) i.e. look how I disable gold linker in systemd.